WcaOnRails/app/controllers/regional_organizations_controller.rb

```diff
@@ -31,6 +31,25 @@ def update
     end
   end
 
+  def destroy
+    @regional_organization = regional_organization_from_params
+    if current_user.can_manage_regional_organizations?
```

Comment:
Why do you check for permissions in the destroy method, whey you could just as easily not exclude it from the general permissions check at line 5?